oc.py

```diff
@@ -67,9 +67,15 @@ def get_pods():
     return parse(oc(cmd))
 
 
-def logs(pod):
-    "Get logs from pod."
+def logs(pod, since=None, tail=None, timestamps=None):
+    "Get logs from pod. Provides flags for the `--since`, `--     tail`, and `--timestamps` flags.."
```

Comment:
I prefer to hide the dash-dash thing, to look more like arguments, not options of command.
Rewrite to `Provide options for since, tail and timestamps`.